### PR TITLE
Added Releasing guidelines

### PR DESCRIPTION
**Pull request by rchampourlier**
Added detailed guidelines on the Releasing process (:package: prepare, :shipit: deploy, :up: update JIRA).

Please have a look :eyes: at it to ensure I've missed nothing and it's clear enough (for experienced as well as beginners).

Thanks! :heart: :heart: :heart:

### CODE REVIEW

**Comment by giniouxe** [comment]
:+1:

**Comment by mru2** [comment]
:+1:

**Comment by pinouchon** [comment]
:+1:

**Comment by Poilon** [comment]
:+1:

File: process/releasing.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Releasing
+
+## Prepare
+
+- **Check with the team**:
+  - if someone as an issue it would like to include to the release.
```
Comment (bakster) [inline process/releasing.md]: je pense que tu voulais dire
if someone **has** an issue **they** would like to....

**Comment by rchampourlier** [comment]
@bakster Merci, c'est corrigé !

**Comment by bakster** [comment]
:+1:

**Pull request merged**